precompiles/werc20/testdata/WEVMOS9.sol

```diff
@@ -20,17 +20,17 @@
 pragma solidity >=0.4.22 <0.6;
 
 contract WEVMOS9 {
-    string public name     = "Wrapped Evmos";
-    string public symbol   = "WEVMOS";
-    uint8  public decimals = 18;
+    string public name = "Wrapped Evmos";
```

Comment:
why is called WEVMOS9?

Reply:
Because this implementation of the wrapped token comes from the standard implementation of WETH, which contract is called WETH9: https://etherscan.io/token/0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2#code

No specific reason, we can also rename it, but this way is clear which is the source contract.